privateca/certificate_authority_subordinate/main.tf

```diff
@@ -51,12 +50,14 @@ resource "google_privateca_certificate_authority" "root_ca" {
   key_spec {
     algorithm = "RSA_PKCS1_4096_SHA256"
   }
+  // Root CA should be valid indefinetly e.g. 99 years
```

Comment:
same as above, please update the comment and value to be 10 years, and avoid the implication of indefinite validity